### ISSUE TITLE
Plotting Activation Functions

### ISSUE THREAD

**Issue by ccarr1er** [zavareh1/Wav-KAN#1]
Is it possible to plot the learned activation functions of Wav-KAN similar to the plot function of KAN?

**Comment by zavareh1** [comment]
Yes; for sure. I had already the plan to add this to the next version of the code and/or paper.
